src/javascript/app/pages/trade/price.js

```diff
@@ -355,21 +345,44 @@ const Price = (() => {
 
         processForgetProposalOpenContract();
         processForgetProposals().then(() => {
+            const position_is_visible = {
+                top   : false,
+                middle: false,
+                bottom: false,
+            };
+            let first_price_proposal = true;
             Object.keys(types || {}).forEach((type_of_contract) => {
```

Comment:
can't we use the index in foreach to detect first price proposal instead?

Reply:
@negarn price proposal is a subscription call here

Reply:
what i mean by `first_price_proposal` is the very first response we get from `proposal: 1`